src/basic/log.h

```diff
@@ -9,6 +9,11 @@
 
 #include "macro.h"
 
+/* Detects a log syntax issue. Having the variable be global for now seems appropriate since there
+ * are more than 800 different log_syntax checks which along with the function calling the parsing
+ * function ignore the return code, so refactoring this entirely would take quite a lot of time */
```

Comment:
Maybe:
> The number of issues logged with log_syntax(). We use a global variable because those errors are usually ignored by the caller, and this way we can know how many issues (if any) were detected without refactoring all the callers.